src/steps/run.go

```diff
@@ -63,7 +63,11 @@ func Run(runState *RunState, repo *git.ProdRepo, driver drivers.CodeHostingDrive
 				exitWithMessages(runState.UnfinishedDetails.CanSkip)
 			}
 		}
-		runState.UndoStepList.Prepend(step.CreateUndoStep())
+		undoSteps, err := step.CreateUndoStep(repo)
```

Comment:
`undoSteps` should be renamed to `undoStep` as it is singular

Reply:
done